+++
title = "Namco: A microbiome explorer"
date = 2022-08-02
authors = ["alexander_dietrich", "monica_matchado", "Maximilian Zwiebel", "Benjamin Ã–lke", "michael_lauber", "Ilias Lagkouvardos", "Jan Baumbach", "Dirk Haller", "Beate Brandl", "Thoams Skurk", "Hans Hauner", "Sandra Reitmeier", "markus_list"]
#type:
#    0 = Uncategorized
#    1 = Conference paper
#    2 = Journal article
#    3 = Preprint / Working Paper
#    4 = Report
#    5 = Book
#    6 = Book section
#    7 = Thesis (v4.2+ required)
#    8 = Patent (v4.2+ required)
publication_types = ["2"]
publication = "*Microbial Genomics*"

# Abstract and optional shortened version.
abstract = "Background: 16S rRNA gene profiling is currently the most widely used technique in microbiome research and allows for studying microbial diversity, taxonomic profiling, phylogenetics, functional and network analysis. While a plethora of tools have been developed for the analysis of 16S rRNA gene data, only few platforms offer a user-friendly interface and none comprehensively cover the whole analysis pipeline from raw data processing down to complex analysis. Results: We introduce Namco, an R shiny application that offers a streamlined interface and serves as a one-stop solution for microbiome analysis. We demonstrate Namco's capabilities by studying the association between rich fibre diet and the gut microbiota composition. Namco helped to prove the hypothesis that butyrate-producing bacteria are prompted by fibre-enriched intervention. Conclusion: Namco provides a broad range of features from raw data processing and basic statistics down to machine learning and network analysis, thus covering complex data analysis tasks that are not comprehensively covered elsewhere. Namco is freely available at https://exbio.wzw.tum.de/namco/."

# Featured image thumbnail (optional)
image_preview = ""

# Is this a selected publication? (true/false)
selected = false

# Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter your project's filename without extension.
#   E.g. `projects = ["deep-learning"]` references `content/project/deep-learning.md`.
#   Otherwise, set `projects = []`.
projects = []

# Tags (optional).
#   Set `tags = []` for no tags, or use the form `tags = ["A Tag", "Another Tag"]` for one or more tags.
tags = ["microbiome"]

# Links (optional).
url_pdf = "https://www.microbiologyresearch.org/content/journal/mgen/10.1099/mgen.0.000852"
url_preprint = "https://www.biorxiv.org/content/10.1101/2021.12.15.471754v1"
url_code = "https://github.com/biomedbigdata/namco"
url_dataset = "https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA774891"
url_project = "https://exbio.wzw.tum.de/namco/"
url_slides = ""
url_video = "https://www.youtube.com/watch?v=dMx2nmXqMfU"
url_poster = ""
url_source = ""
doi = "10.1099/mgen.0.000852"

# Custom links (optional).
#   Uncomment line below to enable. For multiple links, use the form `[{...}, {...}, {...}]`.
# url_custom = [{name = "Custom Link", url = "http://example.org"}]

# Does this page contain LaTeX math? (true/false)
math = false

# Does this page require source code highlighting? (true/false)
highlight = true

+++
