+++
title = "Looking at protein-protein interaction networks from the perspective of alternative splicing"
publishDate = 2021-09-02T09:30:00+01:00  # Schedule page publish date.
date = 2021-10-19T14:00:00+01:00  # Schedule page publish date.
#date_end = 2021-11-02T10:30:00+01:00
draft = false
authors = ["markus_list"]

# Talk start and end times.
#   End time can optionally be hidden by prefixing the line with `#`.
#time_start = 2021-11-02T9:30:00+01:00
#time_end = 2019-09-27T13:35:00+01:00

# Abstract and optional shortened version.
abstract = "Protein-protein interaction networks have been used since the early days of systems biology to model functional relationships between genes and proteins. On the computational side, they have been immensely helpful for de novo network enrichment strategies that could uncover novel disease modules as potential mechanisms or as a source of biomarkers and drug targets. In recent years, it has become clear though that protein-protein interaction networks have limits, e.g. due to literature and experimental bias as well as lack of cell-type or tissue-specificity that also affects the results of network enrichment methods that rely on them. Widely available next-generation sequencing data nowadays offers the possibility to refine these networks to account for context-specific changes that reflect cell-type, tissue-, disease-specific or even individual variations of the global interactome. Information about domain-domain interactions and alternative splicing can be used to refine protein-protein interaction networks. I will present our database DIGGER as well as the functional enrichment tool NEASE which together offer unique insights into alternative splicing changes on a network level."

abstract_short = ""

# Name of event and optional event URL.
event = "IEM Seminar, Lehrstuhl und Institut für Umweltmedizin, Helmholtz Zentrum München und Universität Augsburg"
event_url = "https://www.compsysmed.de/symdroid/"

# Location of event.
location = " virtual"

# Is this a selected talk? (true/false)
selected = true

# Projects (optional).
#   Associate this talk with one or more of your projects.
#   Simply enter your project's filename without extension.
#   E.g. `projects = ["deep-learning"]` references `content/project/deep-learning.md`.
#   Otherwise, set `projects = []`.
projects = [] 

# Tags (optional).
#   Set `tags = []` for no tags, or use the form `tags = ["A Tag", "Another Tag"]` for one or more tags.
#tags = ["systems medicine", "network medicine"]

# Links (optional).
url_pdf = ""
url_slides = ""
url_video = ""
url_code = ""

# Does the content use math formatting?
math = false

# Does the content use source code highlighting?
highlight = true

# Featured image
# Place your image in the `static/img/` folder and reference its filename below, e.g. `image = "example.jpg"`.
[header]
#image = "deepblueR.png"
#caption = "DeepBlue epigenomic data server"

+++
