+++
title = "Analysing large-scale epigenomic data"
date = 2018-11-06  # Schedule page publish date.
draft = false

# Talk start and end times.
#   End time can optionally be hidden by prefixing the line with `#`.
time_start = 2019-12-12T16:00:00+02:00
time_end = 2019-12-12T16:00:00+02:00

# Abstract and optional shortened version.
abstract = "Large amounts of epigenomic data are publicly available, yet their retrieval for downstream analysis is a research bottleneck. Typically, users download huge files that span the entire genome, even if they are only interested in a small subset (e.g. promoter regions) or an aggregation thereof. Moreover, complex operations on genome-level data are not always feasible on a local computer due to resource limitations.</br></br>The DeepBlue Epigenomic Data Server (http://deepblue.mpi-inf.mpg.de/) mitigates this issue by providing a powerful interface and API for filtering, transforming, aggregating and downloading multi-scale data from several epigenomic consortia, making it the ideal resource for integrating epigenomics resources into analysis workflows and tools.</br></br>While any programming language can be used to access the DeepBlue API, we developed an R/Bioconductor package (https://doi.org/doi:10.18129/B9.bioc.DeepBlueR) that enables users not proficient in scripting or programming languages to analyze epigenomic data in a user-friendly way. Here, the extracted data are automatically converted to suitable R data structures for downstream analysis and visualization within the Bioconductor framework.</br></br>In the second part of my talk, I will highlight our recent work on DNA methylation heterogeneity on bulk bisulfite sequencing data. Notably, most studies of DNA methylation focus almost exclusively on mapping differences in average methylation, neglecting the contribution of cell-to-cell variability. While various scores have been proposed to capture DNA methylation heterogeneity, we performed the first systematic comparison of existing scores on both simulated and experimental data. In our benchmark, we consider different scenarios from which heterogeneity can arise, including cell-type heterogeneity and allele-specific methylation, and propose two new scores that accurately capture DNA methylation heterogeneity and are the first to cover individual CpGs."

abstract_short = ""

# Name of event and optional event URL.
event = "QBiC TechTalks"
event_url = "https://uni-tuebingen.de/forschung/forschungsinfrastruktur/zentrum-fuer-quantitative-biologie-qbic/veranstaltungen/techtalks/"

# Location of event.
location = "Auf der Morgenstelle 16, 72076 TÃ¼bingen, Germany"

# Is this a selected talk? (true/false)
selected = true

# Projects (optional).
#   Associate this talk with one or more of your projects.
#   Simply enter your project's filename without extension.
#   E.g. `projects = ["deep-learning"]` references `content/project/deep-learning.md`.
#   Otherwise, set `projects = []`.
projects = ["DeepBlue"] 

# Tags (optional).
#   Set `tags = []` for no tags, or use the form `tags = ["A Tag", "Another Tag"]` for one or more tags.
tags = ["epigenomics", "bioconductor"]

# Links (optional).
url_pdf = ""
url_slides = ""
url_video = ""
url_code = ""

# Does the content use math formatting?
math = false

# Does the content use source code highlighting?
highlight = true

# Featured image
# Place your image in the `static/img/` folder and reference its filename below, e.g. `image = "example.jpg"`.
[header]
image = "deepblueR.png"
caption = "DeepBlue epigenomic data server"

+++
