+++
title = "Alternative splicing analysis benchmark with DICAST"
date = 2022-01-06
authors = ["amit_fenn", "olga_tsoy", "tim_faro", "fanny_roessler", "alexander_dietrich", "johannes_kersting", "zakaria_louadi", "Chit Tong Lio", "Uwe Voelker", "Jan Baumbach", "Tim Kacprowski", "markus_list"]
#type:
#    0 = Uncategorized
#    1 = Conference paper
#    2 = Journal article
#    3 = Preprint / Working Paper
#    4 = Report
#    5 = Book
#    6 = Book section
#    7 = Thesis (v4.2+ required)
#    8 = Patent (v4.2+ required)
publication_types = ["3"]
publication = "*bioRxiv*"

# Abstract and optional shortened version.
abstract = "Alternative splicing is a major contributor to transcriptome and proteome diversity in health and disease. A plethora of tools have been developed for studying alternative splicing in RNA-seq data. Previous benchmarks focused on isoform quantification and mapping. They neglected event detection tools, which arguably provide the most detailed insights into the alternative splicing process. DICAST offers a modular and extensible framework for the analysis of alternative splicing integrating 11 splice-aware mapping and eight event detection tools. We benchmark all tools extensively on simulated as well as whole blood RNA-seq data. STAR and HISAT2 demonstrated the best balance between performance and run time. The performance of event detection tools varies widely with no tool outperforming all others. DICAST allows researchers to employ a consensus approach to consider the most successful tools jointly for robust event detection. Furthermore, we propose the first reporting standard to unify existing formats and to guide future tool development."

# Featured image thumbnail (optional)
image_preview = ""

# Is this a selected publication? (true/false)
selected = false

# Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter your project's filename without extension.
#   E.g. `projects = ["deep-learning"]` references `content/project/deep-learning.md`.
#   Otherwise, set `projects = []`.
projects = []

# Tags (optional).
#   Set `tags = []` for no tags, or use the form `tags = ["A Tag", "Another Tag"]` for one or more tags.
tags = ["microbiome"]

# Links (optional).
url_pdf = "https://www.biorxiv.org/content/10.1101/2022.01.05.475067v1.full.pdf"
url_preprint = ""
url_code = "https://github.com/CGAT-Group/DICAST "
url_dataset = ""
url_project = "https://dicast.readthedocs.io/en/master/contents.html"
url_slides = ""
url_video = ""
url_poster = ""
url_source = ""
doi = "10.1101/2022.01.05.475067"

# Custom links (optional).
#   Uncomment line below to enable. For multiple links, use the form `[{...}, {...}, {...}]`.
# url_custom = [{name = "Custom Link", url = "http://example.org"}]

# Does this page contain LaTeX math? (true/false)
math = false

# Does this page require source code highlighting? (true/false)
highlight = true

+++
