+++
title = "Arrhythmic gut microbiome signatures for risk profiling of Type-2 Diabetes"
date = 2019-01-01
authors = ["Sandra Reitmeier", "Silke Kiessling", "Thomas Clavel", "Markus List", "Eduardo L Almeida", "Tarini G Ghosh", "Klaus Neuhaus", "Martina Troll", "Harald Grallert", "Wolfgang Rathmann", " others"]
publication_types = ["7"]
abstract = ""
selected = false
publication = "*bioRxiv*"
doi = "10.1101/2019.12.27.889865"
+++

# Abstract and optional shortened version.
abstract = "To combat the epidemic increase in Type-2-Diabetes (T2D), risk factors need to be identified. Diet, lifestyle and the gut microbiome are among the most important factors affecting metabolic health. We demonstrate in 1,976 subjects of a prospective population cohort that specific gut microbiota members show diurnal oscillations in their relative abundance and we identified 13 taxa with disrupted rhythmicity in T2D. Prediction models based on this signature classified T2D with an area under the curve of 73%. BMI as microbiota-independent risk marker further improved diagnostic classification of T2D. The validity of this arrhythmic risk signature to predict T2D was confirmed in 699 KORA subjects five years after initial sampling. Shotgun metagenomic analysis linked 26 pathways associated with xenobiotic, amino acid, fatty acid, and taurine metabolism to the diurnal oscillation of gut bacteria. In summary, we determined a cohort-specific risk pattern of arrhythmic taxa which significantly contributes to the classification and prediction of T2D, highlighting the importance of circadian rhythmicity of the microbiome in targeting metabolic human diseases."

# Featured image thumbnail (optional)
image_preview = ""

# Is this a selected publication? (true/false)
selected = false

# Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter your project's filename without extension.
#   E.g. `projects = ["deep-learning"]` references `content/project/deep-learning.md`.
#   Otherwise, set `projects = []`.
projects = []

# Tags (optional).
#   Set `tags = []` for no tags, or use the form `tags = ["A Tag", "Another Tag"]` for one or more tags.
tags = ["microbiome", "16S rRNA amplicon sequencing", "circadian rhythm", "machine learning", "diabetes"]

# Links (optional).
url_pdf = "https://www.biorxiv.org/content/10.1101/2019.12.27.889865v1.full.pdf"
url_preprint = "https://www.biorxiv.org/content/10.1101/2019.12.27.889865v1"
url_code = ""
url_dataset = "https://www.helmholtz-muenchen.de/kora/index.html"
url_project = ""
url_slides = ""
url_video = ""
url_poster = ""
url_source = ""
doi = "10.1101/2019.12.27.889865"

# Custom links (optional).
#   Uncomment line below to enable. For multiple links, use the form `[{...}, {...}, {...}]`.
# url_custom = [{name = "Custom Link", url = "http://example.org"}]

# Does this page contain LaTeX math? (true/false)
math = false

# Does this page require source code highlighting? (true/false)
highlight = true