+++
title = "EpiRegio: analysis and retrieval of regulatory elements linked to genes"
date = 2020-05-27
authors = ["Nina Baumgarten", "Dennis Hecker", "Sivarajan Karunanithi", "Florian Schmidt", "markus_list", "Marcel H Schulz"]
publication_types = ["2"]
publication = "*Nucleic Acids Research*"

# Abstract and optional shortened version.
abstract = "A current challenge in genomics is to interpret non-coding regions and their role in transcriptional regulation of possibly distant target genes. Genome-wide association studies show that a large part of genomic variants are found in those non-coding regions, but their mechanisms of gene regulation are often unknown. An additional challenge is to reliably identify the target genes of the regulatory regions, which is an essential step in understanding their impact on gene expression. Here we present the EPIREGIO web server, a resource of regulatory elements (REMs). REMs are genomic regions that exhibit variations in their chromatin accessibility profile associated with changes in expression of their target genes. EPIREGIO incorporates both epigenomic and gene expression data for various human primary cell types and tissues, providing an integrated view of REMs in the genome. Our web server allows the analysis of genes and their associated REMs, including the REM’s activity and its estimated cell type-specific contribution to its target gene’s expression. Further, it is possible to explore genomic regions for their regulatory potential, investigate overlapping REMs and by that the dissection of regions of large epigenomic complexity. EPIREGIO allows programmatic access through a REST API and is freely available at https://epiregio.de/."

# Featured image thumbnail (optional)
image_preview = ""

# Is this a selected publication? (true/false)
selected = false

# Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter your project's filename without extension.
#   E.g. `projects = ["deep-learning"]` references `content/project/deep-learning.md`.
#   Otherwise, set `projects = []`.
projects = []

# Tags (optional).
#   Set `tags = []` for no tags, or use the form `tags = ["A Tag", "Another Tag"]` for one or more tags.
tags = ["epigenomics"]

# Links (optional).
url_pdf = "https://academic.oup.com/nar/advance-article-pdf/doi/10.1093/nar/gkaa382/33315860/gkaa382.pdf"
url_preprint = ""
url_code = ""
url_dataset = ""
url_project = "https://epiregio.de/"
url_slides = ""
url_video = ""
url_poster = ""
url_source = ""
doi = "10.1093/nar/gkaa382"

# Custom links (optional).
#   Uncomment line below to enable. For multiple links, use the form `[{...}, {...}, {...}]`.
# url_custom = [{name = "Custom Link", url = "http://example.org"}]

# Does this page contain LaTeX math? (true/false)
math = false

# Does this page require source code highlighting? (true/false)
highlight = true

# Featured image
# Place your image in the `static/img/` folder and reference its filename below, e.g. `image = "example.jpg"`.
[header]

+++
