+++
title = "DysRegNet: Patient-specific and confounder-aware dysregulated network inference"
date = 2022-05-01
authors = ["olga_lazareva", "zakaria_louadi", "johannes_kersting","Jan Baumbach", "david_blumenthal", "markus_list"]

#type:
#    0 = Uncategorized
#    1 = Conference paper
#    2 = Journal article
#    3 = Preprint / Working Paper
#    4 = Report
#    5 = Book
#    6 = Book section
#    7 = Thesis (v4.2+ required)
#    8 = Patent (v4.2+ required)
publication_types = ["3"]
publication = "*bioRxiv*"

# Abstract and optional shortened version.
abstract = "Gene regulation is frequently altered in diseases in unique and often patient-specific ways. Hence, personalized strategies have been proposed to infer patient-specific gene-regulatory networks. However, existing methods do not focus on disease-specific dysregulation or lack assessments of statistical significance. Moreover, they do not account for clinically important confounders such as age, sex or treatment history.\n\n To overcome these shortcomings, we present DysRegNet, a novel method for inferring patient-specific regulatory alterations (dysregulations) from gene expression profiles. We compared DysRegNet to state-of-the-art methods and demonstrated that DysRegNet produces more interpretable and biologically meaningful networks. Independent information on promoter methylation and single nucleotide variants further corroborate our results. We apply DysRegNet to eleven TCGA cancer types and illustrate how the inferred networks can be used for down-stream analysis. We show that unique as well as cancer-type-specific dysregulation patterns exist and highlight immune-related mechanisms that are not obvious when focusing on individual genes rather than their interactions.\n\n DysRegNet is available as a Python package (https://github.com/biomedbigdata/DysRegNet_package) and analysis results for eleven TCGA cancer types are further available through an interactive web interface (https://exbio.wzw.tum.de/dysregnet)."

# Featured image thumbnail (optional)
image_preview = ""

# Is this a selected publication? (true/false)
selected = false

# Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter your project's filename without extension.
#   E.g. `projects = ["deep-learning"]` references `content/project/deep-learning.md`.
#   Otherwise, set `projects = []`.
projects = []

# Tags (optional).
#   Set `tags = []` for no tags, or use the form `tags = ["A Tag", "Another Tag"]` for one or more tags.
tags = []

# Links (optional).
url_pdf = "https://www.biorxiv.org/content/10.1101/2022.04.29.490015v1.full.pdf"
url_preprint = "https://www.biorxiv.org/content/10.1101/2022.04.29.490015v1.abstract"
url_code = "https://github.com/biomedbigdata/DysRegNet_package"
url_dataset = ""
url_project = "https://exbio.wzw.tum.de/dysregnet"
url_slides = ""
url_video = ""
url_poster = ""
url_source = ""
doi = "10.1101/2022.04.29.490015"

# Custom links (optional).
#   Uncomment line below to enable. For multiple links, use the form `[{...}, {...}, {...}]`.
# url_custom = [{name = "Custom Link", url = "http://example.org"}]

# Does this page contain LaTeX math? (true/false)
math = false

# Does this page require source code highlighting? (true/false)
highlight = true

+++
